Documentation/Books/Manual/Architecture/DeploymentModes/Cluster/Architecture.md

```diff
@@ -95,9 +101,31 @@ which are suitable for different usage scenarios:
     latency. Essentially, this moves some of the database distribution
     logic to the machine where the client runs.
 
-As you acn see, the _Coordinator_ layer can be scaled and deployed independently
+As you can see, the _Coordinator_ layer can be scaled and deployed independently
 from the _DBServer_ layer.
 
+{% hint 'warning' %}
+It is a best practice and a recommended approach to not run _Agent_ instances
```

Comment:
One can easily miss the **not** in the sentence, let's write it the other way around!
```suggestion
It is a best practice and a recommended approach to run _Agent_ instances
```